docs/iris/src/whatsnew/contributions_v2.0.0/incompatiblechange_2017-Oct-18_rules-removal.txt

```diff
@@ -0,0 +1,12 @@
+The following classes and functions have been removed from :mod:`iris.fileformats.rules`:
+	* ```iris.fileformat.rules.calculate_forecast_period```
+	* ```iris.fileformat.rules.log```
+	* ```iris.fileformat.rules.CMAttribute```
+	* ```iris.fileformat.rules.CMCustomAttribute```
+	* ```iris.fileformat.rules.CoordAndDims```
+	* ```iris.fileformat.rules.DebugString```
+	* ```iris.fileformat.rules.FunctionRule```
+	* ```iris.fileformat.rules.ProcedureRule```
+	* ```iris.fileformat.rules.Rule```
+	* ```iris.fileformat.rules.RulesContainer```
+	* ```iris.fileformat.rules.RuleResult```
```

Comment:
You're missing the removal of the `legacy_custom_rules` keyword arg from the `Loader` class in this module. Can you add a new line to the end of this document:

```
* Removed deprecated keyword argument ```legacy_custom_rules``` from :class:`iris.fileformats.rules.Loader`.
```